assets/src/scripts/components/SearchBar/SearchBarResult.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+/**
+ * History List.
+ */
+export default class SearchBar_Result extends React.Component {
+
+	render() {
+
+		var classes = [ 'SearchBar_Result' ];
```

Comment:
Although in this case... i'm switching to that classNames helper and I think it makes sense to be `let` :)